### DISCUSSION TITLE
Mopeka Sensor with extended bluetooth antenna

### DISCUSSION THREAD

**Discussion by bcrazycramer** [jgyates/genmon#1076] [General]
Would these Mopeka sensors with extended range bluetooth antennas work? 

https://www.amazon.com/dp/B0BKLFQ3CL?ref=emc_s_m_5_i_atc

Thanks,
Brandon

**Comment by jgyates** [comment]
I think it will (assuming this is just a pro model with a larger antenna), but I honestly do not know as I do not have access to one of these and I have no feedback from anyone how may have one. If it does not work it may be as simple as adding a new device ID if the genmon add on for this type of sensor. The existing code supports 8 different sensor IDs that were from a Mopeka integration PDF.
